src/ChildProcessMessageStream.ts

```diff
@@ -0,0 +1,30 @@
+import { BasePostMessageStream, StreamData } from './BasePostMessageStream';
+
+export class ChildProcessMessageStream extends BasePostMessageStream {
+  constructor() {
+    super();
+
+    this._onMessage = this._onMessage.bind(this);
+    globalThis.process.on('message', this._onMessage);
+
+    this._handshake();
+  }
+
+  protected _postMessage(data: StreamData): void {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    globalThis.process.send!({ data });
+  }
+
+  private _onMessage(message: any): void {
```

Comment:
We should use `unknown` here:

```suggestion
  private _onMessage(message: unknown): void {
```